### PR TITLE
Fix MatchReportFormat missing events

### PR DESCRIPTION
**Pull request by lucaxvi**
Added descriptions and icons for new events:

- Injuries (401-425)
- Weather (390, 391)
- Man marking (380-388) - with 3 new icons (14x14 PNGs)

There is one small issue with injuries: I couldn't figure out a way to distinguish "bruised" events from actual injuries since they use the same Event IDs, so both kind of events use the "red cross" icon.

### CODE REVIEW

**Comment by minj** [comment]
This change is [Reviewable](https://reviewable.io/reviews/minj/foxtrick/1605)

**Comment by minj** [comment]
Unfortunately, my plans have changed and I may not have time to look at your PRs this week.

Apologies for the delay.

**Comment by lucaxvi** [comment]
There is one event (id 384) that I forgot to add because it was missing from the official CHPP documentation. I reported the issue to Teles and I will add the event here as soon as he provides an event description in the documentation.

File: content/foxtrick.properties [hunk 1]
```diff
@@ -1223,6 +1277,8 @@ match.events.511=Yellow card cheating
 match.events.512=Red card (2nd warning) nasty play
 match.events.513=Red card (2nd warning) cheating
 match.events.514=Red card without warning
+match.events.597=
```
Comment (minj) [inline content/foxtrick.properties]: missing description

**Comment by minj** [comment]
NOTES:

Missing icons:
```
35-39
75-76
307-309
```
Missing lang/desc: `597-598`

Missing completely:
```
389
450-493
601-604
650-651
```

**Comment by minj** [comment]
Since I was working on this a long time ago, I merged your changes into mine (branch `update-events-1527`)

We have all descriptions already, only icon definitions are missing. Let me know if you are interested in working on it.

**Comment by lucaxvi** [comment]
Hattrick released a new match page in May that basically replicates MatchReportFormat features, I don't think it's necessary to work on this anymore.

**Comment by minj** [comment]
That may be but I still would like to know the source of the images

**Comment by lucaxvi** [comment]
I can't recall.

**Pull request closed without merging**